controllers/admin/AdminStoresController.php

```diff
@@ -566,6 +566,17 @@ public function beforeUpdateOptions()
         }
     }
 
+    public function updateOptionPsShopName($value)
+    {
+        if (!Shop::isFeatureActive() && !$this->errors && $value) {
+            Db::getInstance()->execute(
+                'UPDATE `' . _DB_PREFIX_ . 'shop` ' .
+                'SET name = "' . pSQL($value) . '" ' .
```

Comment:
```suggestion
                'SET name = "' . pSQL($name) . '" ' .
```

Related to my previous comment